src/components/statusLight/StatusLight.css

```diff
@@ -0,0 +1,38 @@
+div .container {
```

Comment:
refactor this with `withStyles`
animation also works: see https://github.com/mui-org/material-ui/blob/353c0aedc50de8bb2c09626e520d7cc16ac461dc/packages/material-ui/src/ButtonBase/TouchRipple.js#L84